metricbeat/module/elasticsearch/node_stats/_meta/fields.yml

```diff
@@ -156,3 +156,154 @@
+      type: object
+      object_type: long
+      description: >
+    - name: thread_pool.*.completed
+      type: object
+      object_type: long
+      description: >
+
+    - name: transport
+      type: group
+      description: >
+        Transport stats
+      fields:
+        - name: rx
+          type: group
+          description: >
+            Received stats
+          fields:
+            - name: count
+              type: long
+              description: >
+            - name: bytes
+              type: long
+              format: bytes
+              description: >
+        - name: tx
+          type: group
+          description: >
+            Transmitted stats
+          fields:
+            - name: count
+              type: long
+              description: >
+            - name: bytes
+              type: long
+              format: bytes
+              description: >
+
+    - name: breakers
+      type: group
+      description: >
+        Circuit breaker stats
+      fields:
+        - name: fielddata
+          type: group
+          fields:
+            - name: limit.bytes
+              type: long
+              format: bytes
+              description: >
+            - name: estimated.bytes
+              type: long
+              format: bytes
+              description: >
+            - name: overhead
+              type: float
+              description: >
+            - name: tripped
+              type: long
+              description: >
+
+    - name: discovery
+      type: group
+      description: >
+        Discover stats
+      fields:
+        - name: cluster_state_queue
+          type: group
+          description: >
+            Cluster state queue stats
+          fields:
+            - name: total
+              type: long
+              description: >
+            - name: pending
+              type: long
+              description: >
+            - name: committed
+              type: long
+              description: >
+        - name: published_cluster_states
+          type: group
+          description: >
+            Published cluster states stats
+          fields:
+            - name: full_states
+              type: long
+              description: >
+            - name: incompatible_diffs
+              type: long
+              description: >
+            - name: compatible_diffs
+              type: long
+              description: >
+
+    - name: ingest
+      type: group
+      description: >
+        Ingest pipeline stats
+      fields:
+        - name: total
+          type: group
+          description: >
+          fields:
+            - name: count
+              type: long
+              description: >
+            - name: current
+              type: long
+              description: >
+            - name: failed
+              type: long
+              description: >
+            - name: ms
+              type: long
+              description: >
+        - name: pipelines.*
```

Comment:
@ruflin I'm not sure the section for this field is generating the correct ES template. Could you take a look at the generated template and and the sample data for this field in `data.json` and tell me how I can generate the correct template?